include/albatross/src/core/dataset.hpp

```diff
@@ -129,4 +155,26 @@ inline auto concatenate_datasets(const RegressionDataset<X> &x,
 
 } // namespace albatross
 
+template <typename X, typename std::enable_if_t<
+                          albatross::is_streamable<X>::value, int> = 0>
+std::ostream &operator<<(std::ostream &os,
+                         const albatross::RegressionDataset<X> &dataset) {
+  for (std::size_t i = 0; i < dataset.size(); ++i) {
+    os << dataset.features[i] << "    " << dataset.targets.mean[i] << "   +/- "
```

Comment:
Is there a separate cout operator definition for the target distribution? Sometimes I print the full covariance, but often I print the mean-diagonal pairing w/o features. It's unclear what would be default. Not important. Writing customized print loops isn't a pain point.

Reply:
Ah I see, yeah I could add one for distributions too.  I was thinking of adding tools like `.top()` and `.bottom()` along the lines of the Eigen methods.  It'd be pretty nice to be able to do:
```
std::cout << dataset.top(5) << std::endl;
```
or
```
std::cout << dataset.targets.top(5) << std::endl;
```
I'll add that in a followup